spellcast.py

```diff
@@ -88,6 +97,19 @@ def __init__(self):
         Reads words from a file and initializes various attributes needed for
         managing the game board.
         """
+        self.words = set()

```

Comment:
`words` is being initialized here as a set but becomes a list on line 114, consider making this a list or changing line 114 to  a set.

Reply:
Fixed and renamed.